akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -144,6 +144,29 @@ akka.serialization.jackson {
     compress-larger-than = 0 KiB
   }
 
+  # Whether the type should be written to the manifest.
+  # If this is off, then either deserialization-type must be defined, or there must be exactly
+  # one serialization binding declared for this serializer, and the type in that binding will be
+  # used as the deserialization type. This feature will only work if that type either is a
+  # concrete class, or if it is a supertype that uses Jackson polymorphism (ie, the
+  # @JsonTypeInfo annotation) to store type information in the JSON itself. The intention behind
+  # disabling this is to remove extraneous type information (ie, fully qualified class names) when
+  # serialized objects are persisted in Akka persistence or replicated using Akka distributed
+  # data. Note that Akka remoting already has manifest compression optimizations that address this,
+  # so for types that just get sent over remoting, this offers no optimization.
+  type-in-manifest = on
```

Comment:
Since this is either `off`, or `on` and requiring `deserialization-type` to be configured as well, could we use only `deserialization-type` as a toggle and have it off when that is empty and on if it contains a class name instead?

Reply:
`deserialization-type` is not required if it's `on`, since if `deserialization-type` is not set, it just looks at the `serialization-bindings`, that's to avoid duplication of config.

Reply:
Ah, I thought about it backwards. NVM then!